src/pygambit/tests/test_behav.py

```diff
@@ -1,5 +1,4 @@
 import unittest
-
```

Comment:
The space between the imports was intentional.  The general pattern is

```
imports of standard library
<blank line>
imports of third-party modules
<blank line>
imports of modules from this package
```

Reply:
Understood, thanks.

Actually, I did not remove a blank link. I included the unittest import, along with inheriting from unittest.TestCase, which was not done only for test_behav.py and test_exten.py when I created my branch. I see that you also fixed this for test_behav.py and moved test_exten.py to pytest.

Reply:
I see in the changelog: "Migrated to pytest for testing of pygambit."  

There's a few that have not been converted: test_behav.py, test_nash.py, test_game_resolve_functions.py, test_file.py.